### PR TITLE
feat: add support for Google Zone Shortname 

### PR DESCRIPTION
**Pull request by r-teller**
- Add support for returning zone shortname
- Add missing region "me-central2"

### CODE REVIEW

**Comment by r-teller** [comment]
@apeabody / @morgante any chance you can approve this?

File: test/fixtures/additional_zones/main.tf [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "example" {
+  source = "../../../examples/additional_zoness"
```
Comment (apeabody) [inline test/fixtures/additional_zones/main.tf]:
```suggestion
  source = "../../../examples/additional_zones"
```

Comment (r-teller) [reply test/fixtures/additional_zones/main.tf]: @apeabody change accepted

**Comment by apeabody** [comment]
Thanks @r-teller!

You should be able to view the lint results in GitHub, here is the relevant section of the INT test:

```
-----> Verifying <additional-zones-gcp>...
[...]
  outputs
     ×  additional zones should return valid short names
     
     expected: "usc2a"
          got: nil
     
     (compared using ==)

     ×  a specific additional_zone shortname should be output
     
     expected: "usc2a"
          got: ""
     
     (compared using ==)
```

File: examples/simple_example/main.tf [hunk 2]
```diff
@@ -15,8 +15,10 @@
  */
 
 module "utils" {
-  source  = "terraform-google-modules/utils/google"
-  version = "~> 0.7"
+  source = "../.."
+  #source  = "terraform-google-modules/utils/google"
+  #version = "~> 0.8"
```
Comment (apeabody) [inline examples/simple_example/main.tf:20]: FYI: This is done automatically by `module-swapper` during the lint and int tests

**Pull request closed without merging**